leejihee/first/과제/information.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
이 페이지에서 제목으로 사용한 h1 태그 내용으로 타이틀을 쓰면 좋을 것 같아요~
```suggestion
    <title>Information</title>
```